tests/bdd/requirements.txt

```diff
@@ -1,4 +1,4 @@
-pytest-bdd
+pytest-bdd==6.0.0
```

Comment:
Should we revert this as this `def _():` seems to be the way forward?
As per - https://github.com/pytest-dev/pytest-bdd/pull/536

Reply:
Let's keep it as is for now, and let this soak, just in case it gets changed upstream..
Then to remove this we should change all our testcode to def _()